Rocket.Chat/Controllers/Chat/ChannelActionsViewController.swift

```diff
@@ -215,7 +218,16 @@ extension ChannelActionsViewController {
     func shareRoom() {
         guard let url = subscription?.externalURL() else { return }
         let controller = UIActivityViewController(activityItems: [url], applicationActivities: nil)
-        present(controller, animated: true, completion: nil)
+
+        if shareRoomCell != nil && UIDevice.current.userInterfaceIdiom == .pad {
+            controller.modalPresentationStyle = .popover
+            controller.popoverPresentationController?.sourceView = shareRoomCell
+            controller.popoverPresentationController?.sourceRect = shareRoomCell.bounds
+        }
+
+        DispatchQueue.main.async {
```

Comment:
Why are you opening another dispatch here?

Reply:
I noticed a delay between the tap on `share room` cell and the activity controller presentation. I thought it might be being called in background so I added this dispatch, and it has improved smoothness.

Reply:
I'm removing it since I double checked and it was already on main thread.